src/PointwiseFunctions/AnalyticSolutions/RelativisticEuler/TovStar.hpp

```diff
@@ -66,6 +66,10 @@ struct DrPressure : db::SimpleTag {
   using type = Scalar<DataType>;
 };
 template <typename DataType>
+struct DcoordsPressure : db::SimpleTag {
```

Comment:
No need to add this tag. Just use `::Tags::deriv<hydro::Tags::Pressure<DataType>, tmpl::size_t<3>, Frame::Inertial>`.